ionic-range-slider.js

```diff
@@ -7,7 +7,7 @@ angular.module("ion.rangeslider", []);
 angular.module("ion.rangeslider").directive("ionRangeSlider",
     function($timeout) {
         return {
-            restrict: "E",
+            restrict: "A",
```

Comment:
That change is a big breaking change. Maybe we should bump a little more the version to `1.1` or something like that